docs/UserGuide.adoc

```diff
@@ -87,6 +87,48 @@ Removes 1 egg from the 1st recipe's list of ingredients.
 * `recipe 2 remove i/Milk` +
 Removes milk from the 2nd recipe's list of ingredients.
 
+==== Add a cooking step
+
+This command allows you to add a cooking step to the selected recipe.
+Requires a valid index in the list of recipes and the cooking step.
```

Comment:
I think we don't have to write this? if not we have to do this for many commands :/ Perhaps we can write the constraints in the intro to the commands.

Reply:
I think this is fine for now, and we can change it once we are decided on what the constraints are?